application/class-btm-admin-manager.php

```diff
@@ -103,13 +103,33 @@ public function btm_admin_settings_sub_page(){
 		if( isset( $_POST[ "btm-cron-interval" ] ) ){
 			$interval = $_POST[ "btm-cron-interval" ];
 			if( ctype_digit( $interval ) ){
```

Comment:
@eriktad do we display an error message if user will type non-numeric numbers?

Reply:
The input type is numeric min value 1. I dont use Js validation.Use it?

Reply:
OK, thanks, I think we are good for now.